pages/BusinessPage.js

```diff
@@ -19,8 +19,11 @@ class BusinessPage extends BasePage {
             //labels
         this.bpBusinessNameLabel = "//h1/span[contains(text(), 'the name')]";
         this.bpAddressConfirmationLabel = "//h1[text()= 'Is this your business?']";
+        this.bpPolicyStartLabel = "//h1[text()= 'When do you want your policy to begin?']";
         this.bpEmployeesQuestionLabel = "//div[contains(text(), 'How many of your employees work more than 20 hours a week?')]"
-        this.bpXmodQuestionLabel = "//div[contains(text(), 'Do you have an experience modification factor (xMod)?')]"
```

Comment:
no more xMod question?

Reply:
Changed the name of the variable as per its HTML tag <div> as it is not a label